src/main/java/org/commcare/formplayer/aspects/LoggingAspect.java

```diff
@@ -23,14 +23,16 @@
 
     private final Log log = LogFactory.getLog(LoggingAspect.class);
 
-    @Around(value = "@annotation(org.springframework.web.bind.annotation.RequestMapping) " +
+    @Around(value = "within(org.commcare.formplayer..*) " +
```

Comment:
what is the result of this change?

Reply:
This changes how the aspect so that it only matches methods that are in or below the `org.commcare.formplayer` package. Specifically this should not run on any of the built in Spring views like `/error` since the do not have the expected method arguments.